packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -838,10 +853,10 @@ export class MgtFileList extends MgtTemplatedComponent implements CardSection {
     }
 
     if (provider.state === ProviderState.SignedOut) {
-      this.files = null;
+      this._files = null;
       return;
     }
-    const graph = provider.graph.forComponent(this);
+    this._graph = provider.graph.forComponent(this);
```

Comment:
You should do this initialization in the `constructor` to avoid having to do this again in another block within the class.

Reply:
Actually, I'm now thinking, when the constructor is called, the provider isn't _set_? If that's the case, it makes sense to leave it as is.